android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -246,10 +246,10 @@ public abstract class BraveActivity extends ChromeActivity
     public static final String REWARDS_AC_SETTINGS_URL = "brave://rewards/contribute";
     public static final String BRAVE_AI_CHAT_URL = "chrome-untrusted://chat";
     public static final String REWARDS_LEARN_MORE_URL =
-            "https://brave.com/faq-rewards/#unclaimed-funds";
+            "https://ping-browser.com/faqs-and-help";
     public static final String BRAVE_TERMS_PAGE =
-            "https://basicattentiontoken.org/user-terms-of-service/";
-    public static final String BRAVE_PRIVACY_POLICY = "https://brave.com/privacy/browser/#rewards";
+            "https://ping-browser.com/faqs-and-help";
+    public static final String BRAVE_PRIVACY_POLICY = "https://ping-browser.com/faqs-and-help";
```

Comment:
please change these urls consciously. 

support urls -> https://ping-browser.com/faqs-and-help
privacy policy -> https://ping-browser.com/privacy-policy
terms of use -> https://ping-browser.com/terms-of-use-1

and check for all urls if any new ones need to be added or updated.
also check for email Ids, we will need to update any existing email ids with ours as well